Gruntfile.js

```diff
@@ -212,5 +212,5 @@ module.exports = function (grunt) {
         });
     });
 
-    grunt.registerTask('default', ['browserify2:edge', 'mocha:test', 'watch']);
+    grunt.registerTask('default', ['browserify2:edge', 'watch']);
```

Comment:
So if you have failing tests, you can call `grunt`, compiles and enters the watch, otherwise you have to keep calling grunt to see if the tests passed.